src/classes/CW_CampaignListSettingHelper.cls

```diff
@@ -0,0 +1,134 @@
+/**
+* File Name   :    CW_CampaignListSettingHelper.cls
+* Description :   Class for getting and filling CW_CampaignListSetting__c with default values
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 08/05/2014 Alex Panka	  Initial
+*/
+
+public with sharing class CW_CampaignListSettingHelper {
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.